src/main/java/ch/cern/eos/XrootDBasedFileSystem.java

```diff
@@ -191,9 +193,13 @@ private static void initJXrdCl() throws IOException {
 		try {
 			System.loadLibrary("jXrdCl");
 		} catch (UnsatisfiedLinkError e) {
-			e.printStackTrace();
-			eosDebugLogger.print("failed to load jXrdCl");
-			throw new IOException();
+            try {
+                NarSystem.loadLibrary();
```

Comment:
❤️

Reply:
BUt please rethink it, try catch within try catch?

Reply:
I think that was just laziness. There's no real reason for it